src/main/java/org/gooru/insights/constants/APIConstants.java

```diff
@@ -233,6 +235,8 @@
 	
 	public static final String DEFAULT_TO_MAIL = "default.to.email";
```

Comment:
@ManikandanV 
Any specific reason for using dot as separator?If not can you maintain consistency?
eg : default_to_email